lib/Command/LintCommand.php

```diff
@@ -93,9 +89,54 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
+        // yaml-lint only supports one file at a time
+        $label = 'YAML checks';
+        $succeed = $this->syncFindExec(['find', $dir, '-type', 'f', '-name', '*.yml', '!', '-path', '*/vendor/*'], ['vendor/bin/yaml-lint']);
+
+        if (!$succeed) {
+            $style->section('YAML checks');
+            $style->error("$label failed\n");
+            $exit = $exit | self::ERR_YAML;
+        } else {
+            $style->success("$label successfull\n");
+        }
+
         return $exit;
     }
 
+    private function syncFindExec(array $findCmd, array $execCmd)
```

Comment:
Instead of re-implementing everything, next time something like https://github.com/OndraM/php-coding-standard/commit/c361845e8ee49c2aa7650428a0f62672ab5038f3 could work